doc/classes/Window.xml

```diff
@@ -712,7 +722,7 @@
 			If [code]true[/code], the [Window] can't be focused nor interacted with. It can still be visible.
 		</member>
 		<member name="unresizable" type="bool" setter="set_flag" getter="get_flag" default="false">
-			If [code]true[/code], the window can't be resized. Minimize and maximize buttons are disabled.
+			If [code]true[/code], the window can't be resized.
```

Comment:
I don't believe many will be bothered by this change. But, because this is a change in behaviour nonetheless, it does break user's expectations. So... it breaks compatibility?

Reply:
Yes. But it was not working as documented, only maximize button was disabled.